views/partials/header.ejs

```diff
@@ -110,6 +110,75 @@
 </header>
 <!--end header-->
 
+<script>
+    //Modal Popup Controller
+    function toggle_visibility(id){
+        var e = document.getElementById(id);
+        if (e.style.display == 'none') {
+            e.style.display = 'block';
+        }
+        else if (e.style.display == 'block'){
+            e.style.display = 'none'
+        }
+    };
+
+    function startTimer(duration, display) {
+        var timer = duration, minutes, seconds;
+        var e = document.getElementById('sessionAlert');
+        e.innerText = "Session will expire in 5 minutes";
+        setInterval(function () {
+            minutes = parseInt(timer / 60, 10);
+            seconds = parseInt(timer % 60, 10);
+
+            display.textContent = minutes + " minute(s) and " + seconds + " seconds";
+
+            if (--timer < 0) {
+                timer = duration;
+            }
+        }, 1000);
+        toggle_visibility('expiry-warning');
+    };
+
+    function refreshSession() {
+        location.reload();
+    }
+</script>
+
+<script>
+    var originalSessionAge = 1800000; // Session length
```

Comment:
Should these be a variable, like in the application service?

Reply:
It should, but the user service doesn't have THESESSION in the config, so this was a quick win. 

Should be there before going live